### PR TITLE
Center logout message and fix font

### PR DESCRIPTION
**Pull request by msmner**
Closes https://github.com/SoftUni-Internal/exam-systems-issues/issues/442

### CODE REVIEW

**Review by mishogv — DISMISSED** [review]

File: Servers/UI/OJS.Servers.Ui/ClientApp/src/pages/logout/LogoutPage.module.scss [hunk 1]
```diff
@@ -0,0 +1,8 @@
+﻿@use 'src/styles/fonts';
+
+.logout {
+  display: flex;
+  font: fonts.$font-family-lato;
```
Comment (shsimeonova) [inline Servers/UI/OJS.Servers.Ui/ClientApp/src/pages/logout/LogoutPage.module.scss:5]: Order flex properties meaning justify-content: center; to be just under display flex. This way its easier to read

**Review by shsimeonova — CHANGES_REQUESTED** [review]

**Comment by shsimeonova** [comment]
stylelint throws an error; src/pages/logout/LogoutPage.module.scss
 8:1  ✖  Unexpected missing end-of-source newline  no-missing-end-of-source-newline

File: Servers/UI/OJS.Servers.Ui/ClientApp/src/pages/logout/LogoutPage.module.scss (hunk 1 above)
Comment (msmner) [reply Servers/UI/OJS.Servers.Ui/ClientApp/src/pages/logout/LogoutPage.module.scss:5]: If I try to do that, then there is an error with ESLint due to the properties not being in alphabetical order.

Comment (shsimeonova) [reply Servers/UI/OJS.Servers.Ui/ClientApp/src/pages/logout/LogoutPage.module.scss:5]: Okay

**Review by shsimeonova — APPROVED** [review]

**Review by PetarNeshkov — APPROVED** [review]

**Review by shsimeonova — APPROVED** [review]

**Pull request merged**